subsys/net/l2/openthread/openthread.c

```diff
@@ -453,6 +461,24 @@ static enum net_l2_flags openthread_flags(struct net_if *iface)
 	return NET_L2_MULTICAST;
 }
 
+static int openthread_enabled(struct net_if *iface, bool state)
```

Comment:
nit: This should be called `openthread_enable`

Reply:
good point, thanks